blark/transform.py

```diff
@@ -3266,12 +3273,13 @@ def from_lark(
         access: Optional[AccessSpecifier],
         name: lark.Token,
         return_type: Optional[LocatedVariableSpecInit],
+        access_: Optional[AccessSpecifier],
         *args
     ) -> Property:
         *declarations, body = args
         return Property(
             name=name,
-            access=access,
+            access=access or access_,
```

Comment:
So the latter `access_` gets priority. Dumping the code again would result in `access` being dropped, if somehow both were specified. I think that's probably OK.

Reply:
My guess is it would be invalid (and rejected) by twincat if you specify both, I could try

Reply:
The way I implemented it is backwards, actually the second access specifier should be dominant but isn't in the current implementation.